dev/io.openliberty.jakartaee9.internal_fat/fat/src/io/openliberty/jakartaee9/internal/tests/EE9FeatureCompatibilityTest.java

```diff
@@ -359,7 +359,11 @@ private void testCompatibility(String featureName, Set<String> featureSet, Map<S
         int threadCount = Runtime.getRuntime().availableProcessors() - 1;
         if (threadCount > 4) {
             threadCount = 4;
+        } else if (threadCount == 1) {
+            // default to 2 threads so that we don't go over 3 hours in a slow build machine with only two CPUs
+            threadCount = 2;
         }
+        // leaving <= instead of < in case we change back to only one thread in a two CPU environment.
         if (threadCount <= 1) {
             checkFeatures(featureName, new ArrayDeque<String>(featureSet), specialConflicts, errors);
```

Comment:
I believe this if will no longer be entered in any case.

Reply:
It will still be entered in the 1 CPU case.

Reply:
I don't understand why you need to subtract one thread for all other CPU > 2.  Seems more simple if you just took the threadCount to be the number of available processors and don't go over 4.

Reply:
```java
int threadCount = min(Runtime.getRuntime().availableProcessors(), 4);
```

Reply:
I was doing that to not monopolize every CPU on people's systems when running locally, but if you think that is ok, I can do that.

Reply:
Most systems probably have more than 4 cores to them nowadays.  But it is not that important.  Just looking to make it more simple if we can.  I'll leave that up to you.  I'll approve.